packages/ui/src/components/ToastBar.tsx

```diff
@@ -28,8 +31,8 @@ const ToastBar = ({ toast }: Props) => {
   return (
     <Snackbar
```

Comment:
We are rendering Snackbar inside MuiSnackbar is it expected?

Reply:
Yes this is a weird construct, `Snackbar > Stack > Snackbar`, this also looks weird to me, but works without any additional lib (the `notistack` one) so I closed my eyes on it. But maybe/probably the first Snackbar is actually not useful.

Reply:
I managed to remove a Snackbar.